almond/README.md

```diff
@@ -1,10 +1,15 @@
 # Home Assistant Add-on: Almond
 
-[Almond] For Home Servers.
+## The Open-source, Privacy-Preserving Voice Assistant.
 
 ![Supports aarch64 Architecture][aarch64-shield] ![Supports amd64 Architecture][amd64-shield] ![Supports armhf Architecture][armhf-shield] ![Supports armv7 Architecture][armv7-shield] ![Supports i386 Architecture][i386-shield]
 
-The Open, Privacy-Preserving Virtual Assistant.
+[Almond] provides a complete solution to turn your Home Assistant
+into a smart speaker. It supports playing music, controlling your
+IoTs, checking the weather, and other simple questions.
+
+To use Almond, you should connect a microphone and speaker to the
```

Comment:
Right now, the add-on doesn't require that. Does that change?

Reply:
Yes, the new addon is quite different than the old one, and uses the audio directly.

Reply:
So it cannot be used without? That would be a big breaking change.

Reply:
It can be used without audio, it's just not very useful. I can clarify this in the readme. Do you have a suggested wording?

Reply:
```suggestion
To use Almond as a voice assistant, you should connect a microphone and speaker to the
```